core/src/main/java/org/apache/brooklyn/core/mgmt/persist/DeserializingClassRenamesProvider.java

```diff
@@ -37,7 +37,8 @@
 public class DeserializingClassRenamesProvider {
 
     public static final String DESERIALIZING_CLASS_RENAMES_PROPERTIES_PATH = "classpath://org/apache/brooklyn/core/mgmt/persist/deserializingClassRenames.properties";
-    
+    public static final String KARAF_DESERIALIZING_CLASS_RENAMES_PROPERTIES_PATH = "file:etc/deserializingClassRenames.properties";
+
```

Comment:
Filename should be namespaced `org.apache.brooklyn.****.properties` as karaf will add this to config admin